lab-doug/test/showContents-test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var expect = require('chai').expect;
+var showContents = require(__dirname + '/../lib/showContents');
+
+describe('Testing showContents module', function(){
+  describe('Testing showContents()', function(){
+    it('should return "hello bob"', function(){
+      const result = showContents.showContents();
+      /*
```

Comment:
[If you like comments, I think this is a good style guide to follow for formatting comments](https://github.com/airbnb/javascript#comments)